lib/tuples.gd

```diff
@@ -178,3 +178,46 @@ direct product elements families" );
 DeclareOperation( "DirectProductElement", [ IsList ]);
 DeclareOperation( "DirectProductElementNC",
     [ IsDirectProductElementFamily, IsList ]);
+
+
+#############################################################################
+##
+##
+##  <#GAPDoc Label="DirectProductFamily">
+##  <ManSection>
+##  <Func Name="DirectProductFamily" Arg='args'/>
+##
+##  <Description>
+##  <A>args</A> must be a dense list of <Ref Attr="CollectionsFamily"/>
+##  families, otherwise the function raises an error.
+##  <P/>
+##  <Ref Func="DirectProductFamily"/> returns a collections family <C>fam</C>
+##  with the following property:
+##  Each collection <C>coll</C> in <C>fam</C> is a direct product
```

Comment:
Wait, is that true: is each such collection a direct product? Isn't it just a subset of a direct product in general?

Reply:
Yes, that's true. I realize now that my name for this family is misleading. A better name that comes to my mind is `DirectProductElementCollectionsFamily`. In #3477 I introduce a `IsDirectProductDomain`category, where the objects in this family actually are direct products.

Reply:
I corrected and rewrote the documentation. I hope this is also clearer than the first version.